build.go

```diff
@@ -183,7 +185,7 @@ func Build(builder Builder, options ...Option) {
 		config.exitHandler.Error(fmt.Errorf("unable to read platform bindings %s\n%w", ctx.Platform.Path, err))
 		return
 	}
-	logger.Debugf("Platform Bindings: %+v", ctx.Platform.Bindings)
+	numBindings = len(ctx.Platform.Bindings)
```

Comment:
It seems like you could probably move this into the `logBindings` method. I don't see it used outside of that function.

Reply:
👍